scripts/src/checkautomerge/checkautomerge.py

```diff
@@ -0,0 +1,27 @@
+import time
+import sys
+import argparse
+
+import requests
+
+def ensure_pull_request_not_merged(api_url):
```

Comment:
We could use `retrying` for this instead of sleep:
- https://pypi.org/project/retrying/
- https://github.com/openshift-helm-charts/charts/blob/dev/tests/functional/utils.py#L45

Reply:
Shouldn't this function checking the PR is merged..? If so it should be named `ensure_pull_request_merged` right?

Reply:
This is a direct copy of the script from the charts repo so i prefer to leave them in sync and treat the one in charts repo as the master copy. I do mention that it is copied in the README.

Reply:
That makes sense :+1: resolving